build-tools/unix-distribution-setup/unix-distribution-setup.targets

```
@@ -1,4 +1,4 @@
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Build">
```

Since this is a plain `.targets` file, I don't think you need the `Sdk` at the top?

Maybe just do this for now:

```suggestion
<Project>
```